docker-compose.yml

```diff
@@ -1154,6 +1154,7 @@ services:
         gcc_version: ${GCC_VERSION}
         tz: ${TZ}
         r_prune_deps: ${R_PRUNE_DEPS}
+        r_duckdb_dev: ${R_DUCKDB_DEV:-}
```

Comment:
I think that we could actually do this default setting with `R_PRUNE_DEPS` as well and remove it from `.env` if we wanted to.